source/JABHandler.py

```diff
@@ -836,10 +840,12 @@ def enableBridge():
 
 
 def initialize():
-	global bridgeDll, isRunning
+	global bridgeDll, is_64Bit, isRunning

```

Comment:
This is not necessary

```suggestion
	global bridgeDll, isRunning
```